Xamarin.Forms.ControlGallery.iOS/PerformanceTrackerRenderer.cs

```diff
@@ -0,0 +1,999 @@
+﻿using CoreGraphics;
+using Foundation;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using UIKit;
+using Xamarin.Forms;
+using Xamarin.Forms.ControlGallery.iOS;
+using Xamarin.Forms.Controls;
+using Xamarin.Forms.Maps;
+using Xamarin.Forms.Maps.iOS;
+using Xamarin.Forms.Platform.iOS;
+
+[assembly: ExportRenderer(typeof(PerformanceTracker), typeof(PerformanceTrackerRenderer))]
+[assembly: ExportRenderer(typeof(ActivityIndicator), typeof(PerformanceTrackingActivityIndicator))]
+[assembly: ExportRenderer(typeof(BoxView), typeof(PerformanceTrackingBoxView))]
+[assembly: ExportRenderer(typeof(Button), typeof(PerformanceTrackingButton))]
+[assembly: ExportRenderer(typeof(DatePicker), typeof(PerformanceTrackingDatePicker))]
+[assembly: ExportRenderer(typeof(Editor), typeof(PerformanceTrackingEditor))]
+[assembly: ExportRenderer(typeof(Entry), typeof(PerformanceTrackingEntry))]
+[assembly: ExportRenderer(typeof(Image), typeof(PerformanceTrackingImage))]
+[assembly: ExportRenderer(typeof(Label), typeof(PerformanceTrackingLabel))]
+[assembly: ExportRenderer(typeof(ListView), typeof(PerformanceTrackingListView))]
+[assembly: ExportRenderer(typeof(Map), typeof(PerformanceTrackingMap))]
+[assembly: ExportRenderer(typeof(Picker), typeof(PerformanceTrackingPicker))]
+[assembly: ExportRenderer(typeof(ProgressBar), typeof(PerformanceTrackingProgressBar))]
+[assembly: ExportRenderer(typeof(SearchBar), typeof(PerformanceTrackingSearchBar))]
+[assembly: ExportRenderer(typeof(Slider), typeof(PerformanceTrackingSlider))]
+[assembly: ExportRenderer(typeof(Stepper), typeof(PerformanceTrackingStepper))]
+[assembly: ExportRenderer(typeof(Switch), typeof(PerformanceTrackingSwitch))]
+[assembly: ExportRenderer(typeof(TableView), typeof(PerformanceTrackingTableView))]
+[assembly: ExportRenderer(typeof(TimePicker), typeof(PerformanceTrackingTimePicker))]
+[assembly: ExportRenderer(typeof(WebView), typeof(PerformanceTrackingWebView))]
+
+namespace Xamarin.Forms.ControlGallery.iOS
```

Comment:
This is doing what the `PerformanceTrackerRenderer` for Android is doing, but in a more invasive way. I can't do the same trick without having every renderer bubble up to the `PerformanceTrackerRenderer`.